### PR TITLE
ExpressLRS 2.4GHz RX PA RGB ie. Foxeer ELRS 2.4GHz LNA Receiver

### PR DESCRIPTION
**Pull request by SilentZen**
ExpressLRS 2.4GHz RX PA RGB ie. Foxeer ELRS 2.4GHz LNA Receiver

### CODE REVIEW

File: targets/expresslrs.ini [hunk 1]
```diff
@@ -91,6 +91,26 @@ build_flags =
...
+	'-D CFG_TARGET_NAME="ELRS2GPRGB"'
+	'-D CFG_TARGET_FULLNAME="ExpressLRS 2.4GHz RX PA RGB"'
+	-D LORA_PIN_RXEN=9
+	-D LORA_PIN_TXEN=10
+	-D LORA_POWER=-10
+	-D LORA_PIN_RST=16
+	-D IO_LED_PIN=2
+
+[env:expresslrs_rx_2400_PA_RGB_via_WiFi]
+extends = env:expresslrs_rx_2400_PA_via_UART
```
Comment (MUSTARDTIGERFPV) [inline targets/expresslrs.ini:109]:
```suggestion
extends = env:expresslrs_rx_2400_PA_RGB_via_UART

```

File: targets/expresslrs.ini [hunk 2]
```diff
@@ -91,6 +91,26 @@ build_flags =
...
+upload_port = 192.168.4.1
+extra_scripts = compressed_ota.py
+upload_protocol = espota
+
```
Comment (MUSTARDTIGERFPV) [inline targets/expresslrs.ini:113]: You can strip these given the latest OTA changes. :)
```suggestion
```

File: targets/expresslrs.ini [hunk 3]
```diff
@@ -91,6 +91,26 @@ build_flags =
...
 extends = env:expresslrs_rx_2400_PA_via_UART
 
+; 2400MHz with PA/LN/RGB (Example: Foxeer ELRS 2.4GHz LNA Receiver)
+[env:expresslrs_rx_2400_PA_RGB_via_UART]
+extends = env_expresslrs_rx_2400, env_common_2400
+build_flags =
+	${env_expresslrs_rx_2400.build_flags}
+	${env_common_2400.build_flags}
+	'-D CFG_TARGET_NAME="ELRS2GPRGB"'
+	'-D CFG_TARGET_FULLNAME="ExpressLRS 2.4GHz RX PA RGB"'
```
Comment (MUSTARDTIGERFPV) [inline targets/expresslrs.ini:101]:
```suggestion
	'-D CFG_TARGET_NAME="ELRS2GPARGB"'
	'-D CFG_TARGET_FULLNAME="ExpressLRS 2.4GHz RX PA RGB"'

```

File: targets/expresslrs.ini [hunk 4]
```diff
@@ -91,6 +91,26 @@ build_flags =
 [env:expresslrs_rx_2400_PA_via_WiFi]
 extends = env:expresslrs_rx_2400_PA_via_UART
 
+; 2400MHz with PA/LN/RGB (Example: Foxeer ELRS 2.4GHz LNA Receiver)
```
Comment (MUSTARDTIGERFPV) [inline targets/expresslrs.ini:94]:
```suggestion
; 2400MHz with PA/LNA/RGB (Example: Foxeer ELRS 2.4GHz LNA Receiver)

```

**Review by MUSTARDTIGERFPV — CHANGES_REQUESTED** [review]
Thanks for the PR! If you'd make the minor changes suggested that'd be super helpful - can you also confirm that this PR builds against master and runs successfully on your hardware? :)

**Review by MUSTARDTIGERFPV — APPROVED** [review]

**Comment by MUSTARDTIGERFPV** [comment]
Thanks so much, @SilentZen! You should come by the FormationFlight Discord btw: https://discord.gg/npaX3VxQjh

**Comment by SilentZen** [comment]
New to this process, anything more I need to do on this pull request?

**Review by MUSTARDTIGERFPV — APPROVED** [review]

**Comment by MUSTARDTIGERFPV** [comment]
> New to this process, anything more I need to do on this pull request?

Nope, thanks for the comment to remind me to take a look! :D

**Pull request merged**